deploy/kubernetes/releases/csi-digitalocean-dev.yaml

```diff
@@ -117,7 +117,7 @@ provisioner: dobs.csi.digitalocean.com
 ##############################################
 
 kind: StatefulSet
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
```

Comment:
Just realized that the `apps/v1` also requires a [`selector`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#statefulsetspec-v1-apps) under the spec or otherwise `kubectl apply` is going to fail.